rust/connlib/snownet/src/node.rs

```diff
@@ -42,6 +42,16 @@ pub const HANDSHAKE_TIMEOUT: Duration = Duration::from_secs(20);
 
 const MAX_UDP_SIZE: usize = (1 << 16) - 1;
 
+/// How many packets we will at most buffer during the initial connection setup.
+///
+/// The buffer in [`InitialConnection`] might be populated from various sources, i.e. different code path may want to establish a connection to the same server.
+/// Both, IPv4 and IPv6 have a max payload size of 65536.
+/// 500 packets can thus at most occupy 31.25 MB which is acceptable per connection.
```

Comment:
Not a blocker, but this should be clarified to be accurate to all platforms.

On iOS on recent phones we have a hard memory limit of `50MB` for the entire network extension:

https://developer.apple.com/forums/thread/73148?answerId=732711022#732711022

Reply:
Given our MTU, this theoretical IP limit here doesn't apply I think.